tests/test_enterprise/views/test_grant_data_sharing_permissions.py

```diff
@@ -388,10 +394,16 @@ def test_get_course_specific_consent_not_needed(
             enterprise_customer=enterprise_customer,
             granted=True
         )
-        response = self.client.get(
-            self.url + '?course_id=course-v1%3AedX%2BDemoX%2BDemo_Course&next=https%3A%2F%2Fgoogle.com'
-        )
-        assert response.status_code == 404
+
+        with mock.patch('enterprise.views.render') as mock_render:
+            mock_render.return_value = HttpResponse()
+            self.client.get(
+                self.url + '?course_id=course-v1%3AedX%2BDemoX%2BDemo_Course&next=https%3A%2F%2Fgoogle.com',
+                data={
+                    'enterprise_customer_uuid': enterprise_customer.uuid,
```

Comment:
same here - why do we need this now? i worry that it won't reflect the flow users actually go through; we might not be sending this parameter to the view in these cases in the wild.

Reply:
per above comment, i'll change this back as well